models/articles.models.js

```diff
@@ -29,4 +28,18 @@ exports.selectArticles = () => {
     .then(({rows}) => {
         return rows
     })
+}
+
+exports.updateArticle = (params, updateBody) => {
+    const articleId = Number(params.article_id)
+    if(typeof updateBody.inc_votes !== 'number' || !Object.keys(updateBody).includes('inc_votes') || typeof articleId !== 'number' ){
```

Comment:
We also don't need to do this check, as a psql error will be thrown if article_id is not a number. In this case we just need to check the error code and throw the error in app.js